ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -175,13 +180,11 @@ else
 		netman update -i eth0
 	fi
 
-	#
-	# If the airframe has been previously reset SYS_AUTCONFIG will have been set to 1 and other params will be reset on the next boot.
-	#
+	# To trigger a parameter reset during boot SYS_AUTCONFIG was set to 1 before
 	if param greater SYS_AUTOCONFIG 0
 	then
-		# Reset params except Airframe, RC calibration, sensor calibration, flight modes, total flight time, and next flight UUID.
-		param reset_all SYS_AUTOSTART RC* CAL_* COM_FLTMODE* LND_FLIGHT* TC_* COM_FLIGHT*
+		# Reset parameters except airframe, parameter version, RC calibration + flight modes, sensor + airspeed calibration, total flight time, flight UUID
+		param reset_all SYS_AUTOS* SYS_PA* RC* COM_FLTMODE* CAL_* ASPD_S* LND_FLIGHT* TC_* COM_FLIGHT*
```

Comment:
```suggestion
		param reset_all SYS_AUTOS* SYS_PA* RC* COM_FLTMODE* CAL_* ASPD_S* LND_FLIGHT* TC_* COM_FLIGHT_UUID
```

Reply:
`COM_FLI*` saves even more characters! 😄

Reply:
Removed the shortenings. The reason I even considered them is because we had it before that this line got too long to be executed with some additional customizations. Of course there are better solutions like a proper categorization. Shortening with wildcards is just a low hanging fruit.